<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiTreeView</class>
 <widget class="QWidget" name="UiTreeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	min-width:  		20px;
	min-height: 		20px;
	max-width:  	20px;
	max-height: 	20px;
	width:  				20px;
	height: 			20px;
	border-radius: 10px;
	padding:			0px;
}
QPushButton#actionDuplicate, QPushButton#actionAddRoot, QPushButton#actionSave {
	min-width:  		70px;
	max-width:  	70px;
	width:  				70px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="UiTreeViewWidget" name="view">
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="actionSave">
       <property name="toolTip">
        <string>Save</string>
       </property>
       <property name="text">
        <string>SAVE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="actionDuplicate">
       <property name="toolTip">
        <string>Duplicate</string>
       </property>
       <property name="text">
        <string>DUPLICATE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="actionAddRoot">
       <property name="toolTip">
        <string>Add a folder</string>
       </property>
       <property name="text">
        <string>NEW FOLDER</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="actionAdd">
       <property name="toolTip">
        <string>New</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="actionImport">
       <property name="toolTip">
        <string>Import a file</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="actionRemove">
       <property name="toolTip">
        <string>Delete</string>
       </property>
       <property name="text">
        <string>â€“</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UiTreeViewWidget</class>
   <extends>QTreeWidget</extends>
   <header>items/uitreeviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionAdd</sender>
   <signal>released()</signal>
   <receiver>UiTreeView</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>480</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove</sender>
   <signal>released()</signal>
   <receiver>UiTreeView</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport</sender>
   <signal>released()</signal>
   <receiver>UiTreeView</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAddRoot</sender>
   <signal>released()</signal>
   <receiver>UiTreeView</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>UiTreeView</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>released()</signal>
   <receiver>UiTreeView</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDuplicate</sender>
   <signal>released()</signal>
   <receiver>UiTreeView</receiver>
   <slot>action()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>action()</slot>
  <slot>action(QModelIndex)</slot>
 </slots>
</ui>
